app/jobs/geocode_job.rb

```diff
@@ -0,0 +1,9 @@
+class GeocodeJob < ActiveJob::Base
+  def perform(location)
```

Comment:
ActiveJob introduce (and motivated the development of) https://github.com/rails/globalid